src/cache/precomputed_quantities.jl

```diff
@@ -603,25 +749,45 @@ function set_precomputed_quantities!(Y, p, t)
 
                 @. ρaʲu³ʲ_datau³ʲ_data -=
                     (1 / local_geometry_halflevel.J^2) * (
-                        local_geometry_prev_level.J^2 * (
+                        local_geometry_prev_level.J^2 *
+                        ρaʲ_prev_level *
+                        ((
                             ∇p_perturb³_prev_level_data / ρʲ_prev_level +
                             ∇Φ³_prev_level_data *
                             (ρʲ_prev_level - ρ_ref_prev_level) / ρʲ_prev_level
-                        )
+                        ))
+                    )
+                @. ρaʲu³ʲ_datau³ʲ_data +=
```

Comment:
This is a hack I have to make updraft in bomex. I will try to remove it (because we don't want it in the end), but there will probably be no updraft in bomex, or it will be sensitive at least.

Reply:
removed.